src/create-jobs.js

```diff
@@ -8,19 +8,31 @@ import create_additional_jobs from './additional-checks.js';
 import validateAndNormalizeChecks from './validate-and-normalize-checks-from-config.js';
 import parseJsonFile from "./json.js";
 
+const xmlSchema = {
+    phpunit: 'vendor/phpunit/phpunit/phpunit.xsd',
+    phpcs: 'vendor/squizlabs/php_codesniffer/phpcs.xsd',
+    psalm: 'vendor/vimeo/psalm/config.xsd',
+};
+
 /**
  * @param {String} filename
- * @return {Boolean}
+ * @return {function(): Boolean}
  */
 const fileTest = function (filename) {
     return function () {
-        if (fs.existsSync(filename)) {
-            return true;
-        }
-        return false;
+        return fs.existsSync(filename);
+
     };
 };
 
+/**
+ * @param {String} filename
+ * @return {String}
+ */
+const appendDistIfNecessary = function(filename) {
```

Comment:
can we rename this to `appendDistExtensionIfFileExists` instead?

Reply:
yep, I'm always bad with names so happy to yield on this.

Reply:
My guess is the naming suggestion is to be more descriptive so I went with `appendDistIfFilenameDoesntExist` to clarify that the non-dist version is preferred if both exist.